docs/docs/recipes/integrate-logto/next-js.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: Next.js
+---
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import redirectUriFigure from './assets/next-redirect-uri.png';
+import AppNote from './fragments/_app-note.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
+import AssumingUrl from './fragments/_next-assuming-url.md';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+
+# Next.js: Integrate `@logto/next`
+
+<AppNote type="Traditional web" />
+
+## Add Logto SDK as a dependency
+
+<Tabs>
+
+  <TabItem value="npm" label="npm">
+
+```bash
+npm i @logto/next
+```
+
+  </TabItem>
+  <TabItem value="yarn" label="Yarn">
+
+```bash
+yarn add @logto/next
+```
+
+  </TabItem>
+  <TabItem value="pnpm" label="pnpm">
+
+```bash
+pnpm add @logto/next
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Init LogtoClient
+
+Import and use configs to initialize LogtoClient.
+
+```ts
+// libraries/logto.js
+import LogtoClient from '@logto/next';
+
+export const logtoClient = new LogtoClient({
+  appId: '<your-application-id>',
+  endpoint: '<your-logto-endpoint>', // E.g. http://localhost:3001
+  baseUrl: '<your-nextjs-app-base-url>', // E.g. http://localhost:3000
+  cookieSecret: 'complex_password_at_least_32_characters_long',
+  cookieSecure: process.env.NODE_ENV === 'production',
+});
+```
+
+<AssumingUrl />
+
+## Configure sign-in redirect URI
```

Comment:
```suggestion
## Configure sign-in Redirect URI
```

Reply:
In this article, title is treated as a sentence